pwnagotchi/locale/de/LC_MESSAGES/voice.po

```diff
@@ -62,6 +69,13 @@ msgstr "Ich bin sehr traurig..."
 msgid "I'm sad"
 msgstr "Ich bin traurig"
 
+#, fuzzy
+msgid "Leave me alone ..."
+msgstr "Lass mich in Ruhe..."
+
+msgid "I'm mad at you!"
+msgstr "Ich bin wütend auf Dich!"
```

Comment:
This does not really express anger...

Reply:
"Ich bin echt sauer auf Dich!" - a bit more colloquial